index.html

```diff
@@ -62,8 +63,11 @@ <h2>I am a Software Developer</h2>
           <li>
             <a href="#"><i class="fa-brands fa-twitter"></i></a>
           </li>
-        </ul>
+          <li>
+            <a href="#"><i class="fa-solid fa-m"></i></a>
+          </li>
 
+        </ul>
```

Comment:
- [Optional] Kindly, use your social media link instead of `#` because the `a` tag Apart from being semantically correct, will let you connect your social networks with the icons, using href. 👍 👍